src/autoconfig.ts

```diff
@@ -2,12 +2,11 @@ import { TemplateSettings } from './types'
 import autoconfigTemplate from './views/autoconfig.xml'
 import swig from 'swig-templates'
 
+// Mozilla Thunderbird
 export default (settings: TemplateSettings) =>
-  async (req: Request): Promise<Response> => {
+  async (): Promise<Response> => {
     const compiledTemplate = swig.compile(autoconfigTemplate)
-    const output = compiledTemplate({
```

Comment:
This is style churn, and it looks like your IDE isn't respecting the default `prettier` config that's in this project.  FYI, it'll create a noisy changeset if WebStorm isn't following the `prettier` config at a project level`

Reply:
I just enabled prettier on the project. It'll apply on save now.

Reply:
🙇